tinygrad/runtime/lib.py

```diff
@@ -72,6 +83,12 @@ def transfer(cls, x, shape, dtype, **kwargs):
     ret = cls(prod(shape), dtype, **kwargs)
     ret._transfer(x)
     return ret
+  
+  @classmethod
+  def from_buffer(cls, buffer: RawBuffer, p2p, **kwargs):
+    if p2p >= 1 and isinstance(buffer.realized, RawBufferTransfer):
```

Comment:
Also I don't think putting `transfer` to only `RawMallocBuffer` is correct. Maybe move it out to `RawBuffer`?